### PR TITLE
Fix/pre release

### PR DESCRIPTION
**Pull request by houdini91**
Igor slack PR

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -417,7 +420,7 @@ os=$(uname_os)
 arch=$(uname_arch)
 format=$(get_format_name "${os}" "${arch}" "tar.gz")
 download_dir=$(mktemp -d)
-supported_tools="gensbom valint"
+supported_tools="valint gensbom"
```
Comment (igor-scribe) [inline install.sh:423]: why the order was changed?

File: install.sh [hunk 2]
```diff
@@ -455,4 +458,8 @@ for val in ${tools}; do
   else
       log_err "Tool not support, Supported=${supported_tools}"
   fi
+
+    echo -en "\n"
+
+  
 done
```
Comment (igor-scribe) [inline install.sh:465]: github expects a new line in the end of script

**Review by igor-scribe — COMMENTED** [review]
need to be able to actually run it to give better feedback, can you send a compiled binary ?

**Comment by houdini91** [comment]
Its not a binary is simply a script.

```
curl -sSfL https://raw.githubusercontent.com/scribe-security/fix/pre-release/master/install.sh | sh
```

File: install.sh (hunk 1 above)
Comment (houdini91) [reply install.sh:423]: For my own sanity, its faster to debug ...

**Comment by igor-scribe** [comment]
> Its not a binary is simply a script.
> 
> ```
> curl -sSfL https://raw.githubusercontent.com/scribe-security/fix/pre-release/master/install.sh | sh
> ```

sure, too many multi tasking :-(

**Comment by igor-scribe** [comment]
```
igor@igors-test-machine:~$ curl -sSfL https://raw.githubusercontent.com/scribe-security/fix/pre-release/master/install.sh | sh
curl: (22) The requested URL returned error: 404
igor@igors-test-machine:~$ curl -sSfL https://raw.githubusercontent.com/scribe-security/misc/fix/pre-release/install.sh | sh
scribe info Selected, tool=valint, version=latest
scribe info Trying to download, tool=valint, version=latest
scribe info Downloading, Version=0.0.23
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 5309k  100 5309k    0     0  11.9M      0 --:--:-- --:--:-- --:--:-- 11.9M
scribe info installed /home/igor/.scribe/bin/valint
-en

scribe info Selected, tool=gensbom, version=latest
scribe info Trying to download, tool=gensbom, version=latest
scribe info Downloading, Version=2.1.114
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 21.1M  100 21.1M    0     0  31.6M      0 --:--:-- --:--:-- --:--:-- 31.6M
scribe info installed /home/igor/.scribe/bin/gensbom
-en
```
Now it looks better.
what is `-en` that is shown now ?

**Comment by igor-scribe** [comment]
```
2$ ./install.sh --version
Illegal option --
./install.sh: 376: ./install.sh: Bad substitution
```

**Comment by igor-scribe** [comment]
```
$ ./install.sh /?
scribe info Selected, tool=valint, version=latest
scribe info Trying to download, tool=valint, version=latest
scribe info Downloading, Version=0.0.23
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 5309k  100 5309k    0     0  11.6M      0 --:--:-- --:--:-- --:--:-- 11.5M
scribe info installed /home/igor/.scribe/bin/valint
-en

scribe info Selected, tool=gensbom, version=latest
scribe info Trying to download, tool=gensbom, version=latest
scribe info Downloading, Version=2.1.114
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 21.1M  100 21.1M    0     0  30.2M      0 --:--:-- --:--:-- --:--:-- 30.2M
```
how do I get to `usage` prompt?

**Comment by houdini91** [comment]
-h

**Comment by houdini91** [comment]
removed `-en`

**Comment by houdini91** [comment]
Can we merge?

**Pull request merged**